docs/topics/wasm/wasm-debugging.md

```diff
@@ -1,148 +1,114 @@
 [//]: # (title: Debug Kotlin/Wasm code)
 
-> Kotlin/Wasm is [Alpha](components-stability.md). It may be changed at any time.
->
-{style="note"}
+<primary-label ref="beta"/> 
 
-This tutorial demonstrates how to use your browser to debug your [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/)
+This tutorial demonstrates how to use IntelliJ IDEA and the browser 
+to debug your [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/)
 application built with Kotlin/Wasm.
 
 ## Before you start
 
-Create a project using the Kotlin Multiplatform wizard:
-
-1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
-2. On the **New Project** tab, change the project name and ID to your preference. In this tutorial, we set the name to "WasmDemo" and the ID to "wasm.project.demo".
-
-   > These are the name and ID of the project directory. You can also leave them as they are.
-   >
-   {style="tip"}
+1. [Set up your environment for Kotlin Multiplatform development](https://www.jetbrains.com/help/kotlin-multiplatform-dev/quickstart.html#set-up-the-environment).
+2. Follow the instructions to [create a Kotlin Multiplatform project targeting Kotlin/Wasm](wasm-get-started.md#create-a-project).
 
-3. Select the **Web** option. Make sure that no other options are selected.
-4. Click the **Download** button and unpack the resulting archive.
+## Debug in IntelliJ IDEA
```

Comment:
Probably we need to mention somewhere that it will work only if `JavaScript Debugger` plugin is installed. Then better to have some instruction for that